ts-defs/index.d.ts

```diff
@@ -1005,6 +1024,10 @@ interface TestController {
      */
     getNativeDialogHistory(): Promise<NativeDialogHistoryItem[]>;
     /**
+     * TODO: Returns a collection of browser console messages.
+     */
```

Comment:
```
    /**
     * Returns an object that contains messages output to the browser console.
     *
     * This object exposes the following properties.
     *
     * -- `log` contains log messages written by the console.log() method
     * -- `error` contains error messages written by the console.error() method
     * -- `warn` contains warning messages written by the console.warn() method
     * -- `info` contains information messages written by the console.info() method
     */
```

Reply:
Do we need to list the return object's properties here? If not, just remove them.